rskj-core/src/main/resources/config/testnet.conf

```diff
@@ -51,6 +51,19 @@ peer {
     networkId = 8100
 }
 
+rest {
```

Comment:
Instead of rest I would use a generic name. Here you are providing a service or services in the future and the REST server is the channel or the way to provide then and not the other way. 
I was thinking in something like: 
`management-service {
    modules {
        health-check {
            enabled = false
        }
    }
    rest-config {
        bind_address = localhost
        port = 4446        
    }
}`

So the config from the modules and the rest server would be separated.

Reply:
We decided to go with the `REST` name to group all modules that will be accessed via their `REST API`, so they're, in fact, as their name implies, `REST` modules. Configuration reflects that fact.